.gitignore

```diff
@@ -9,3 +9,4 @@ __pycache__/
 /tools/out/
 /CMakeLists.txt.user
 /*.log
+/.idea

```

Comment:
This line is probably unrelated.
But given there's already `.vs/` and `.vscode/`, would it be better to add `/.idea` here? @StephanTLavavej

Reply:
I think so

Reply:
What IDE is emitting this - CLion? A web search turned up IntelliJ IDEA which seems unlikely for C++ code.

I don't object to a one-line drive-by addition but I'd like to understand what it's for, and the PR description doesn't say.

Reply:
As this line was removed later, it didn't seem related to this PR, I opened #4678 to track this.

Reply:
Great, thank you.